zproject/default_settings.py

```diff
@@ -199,29 +199,47 @@
 EMBEDDED_BOTS_ENABLED = False
 
 DEFAULT_RATE_LIMITING_RULES = {
+    # Applies to API requests in authenticated contexts,
+    # each user gets their own bucket.
     "api_by_user": [
         (60, 200),  # 200 requests max every minute
     ],
+    # Applies to API requests in unauthenticated contexts
+    # (web public views) - ONLY if the request is not authenticated.
     "api_by_ip": [
         (60, 100),
     ],
+    # Applies to requests to our push bouncer API, by an authenticated
+    # remote server.
     "api_by_remote_server": [
         (60, 1000),
     ],
+    # Limits how many authentication attempts with login+password can be made
+    # to a single username. This applies to the authentication backends where login+password
+    # gets submitted to the Zulip server - the ldap backend or regular email+password backend.
```

Comment:
This says what it _does_ apply to -- do we want to give an example of what it _doesn't_ apply to (social auth, I assume?)

Reply:
Yeah, it just doesn't make sense when you're authing and thus failing/succeeding outside of Zulip. Added elaboration.